Makefile

```diff
@@ -149,6 +149,7 @@ create-cluster:
 
 deploy:
 	until svcat version | grep -m 1 'Server Version: v' ; do sleep 1 ; done
+	kubectl create namespace minibroker
```

Comment:
Does this error out if the namespace already exists?

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/56/commits/f54a6fcfb91abf9a3824ead57c03b296d3ae9317.